## Brief summary of Coll et al, 2018

According to WHO 2019 reports, there were **10 million cases** of tuberculosis(TB) and more than a million people lost their lives to this deadly infectious disease. TB is caused by the bacteria, Mycobactrium tuberculosis which affects the lungs of infected individuals and spreads through air, very much like the coronavirus we are currently dealing with. Although multiple treatments are available, the prevelance of resistant TB has created a public health crisis with need for research in this area. Despite the growing resistance, there were no sensitive molecular tests prior to the publication of this data, due to lack of knowledge on the genetic determinants. In this paper, the authors address this issue by characterizing these determinants of resistance to common drugs used for treatment of TB. Focus was on the bacteria exhibiting either of the two types of resistace, **Multidrug-resistant tuberculosis (MDR-TB)**, caused by bacteria that do not respond to Isoniazid and Rifampicin and **extensively drug-resistant tuberculosis (XDR-TB)**, caused by resistance to fluoroquinolones and any of the injectable drugs (Amikacin, Kanamycin or Capreomycin). 

In this paper, the researchers carried out whole genome analysis(GWAS) of 6465 clinical isolates from 30 different geographic locations encompassing four Mtb complex lineages to identify nucleotide variation and loci responsible for resistance against 14 drugs. Sequence data for 2637 isolates among the total was collected using Illumina sequencing by generating paired-end reads of at least 50 bp with at least 50-fold genome coverage. The remaining 3828 isolates were obtained from public domains. The sequence data from all 6465 isolates was then compared with the H37Rv reference genome and used to call SNPs and indels. They observed the presence of high quality genome wide SNPs, indels and large deletions across all samples. During this process, the authors used a number of quality control steps, mentioned in their methods in detail to ensure good quality calls. 93% of the SNPs identified had rare minor alleles (allele frequency <1%). Also, small indels were rare and ranged in size from 1 to 45 bp. Phenotypic analysis of susceptibility to antituberculosis drugs found that 31.2% of isolates were resistant to at least one drug, with 15.1% categorized as MDR-TB and 4.3% categorized as XDR-TB. To identify the genetic loci associated with drug resistance, the authors used a tree-based ‘PhyC’ test for convergent evolution to detect homoplastic variants and a GWAS approach within a mixed-regression framework. The gene-based GWAS of MDR-TB versus susceptible isolates revealed loci, rpoB, the Rv1482c–fabG1 opero, inhA, katG and oxyR´–ahpC. Mutations were most frequently observed in the katG gene that is associated with Isoniazid leading to MDR-TB. Following katG were rpoB (RIF), embB (EMB), and rpsL (STM) whose occurance was synonymous with the history of their usage. On the other hand, the gene-based GWAS of XDR-TB versus MDR-TB revealed mutations in gyrA, rrs, the embC–embA intergenic region and ubiA. Following this, the authors also carried out a GWAS study to identify if there are any lineage specific loci associated with drug resistance. The analyses revealed that associations with the same loci were observed in more than one lineage, the most frequent being Lineage 4. Thus, the GWAS approach was effective at detecting known and unknown genetic determinants of drug resistance and correlation between the loci. 
